### ISSUE TITLE
pkill issue

### ISSUE THREAD

**Issue by macerg** [termux/termux-packages#6768]
pkill no longer kills tasks, it just lists them.

**Comment by xeffyr** [comment]
Something wrong on your side.

`pkill` works as expected.

![Screenshot_20210508-094049](https://user-images.githubusercontent.com/25881154/117529671-7deb3100-afe1-11eb-8d99-fe4543c20a6a.png)

**Comment by Neo-Oli** [comment]
@xeffyr I think this is actually a valid issue. Since `3.3.17` of proceps `pkill` (once again: https://github.com/termux/termux-packages/issues/192) behaves like `pgrep` I think it's because `packages/procps/pgrep.c.patch` was removed in e62dccd1a4d521bbeb5e066fccf564826e79070f by @Grimler91. 

```
~ $ sleep inf &
[1] 19496
~ [1] $ pkill sleep
19496
~ [1] $ ps
  PID TTY          TIME CMD
18249 pts/10   00:00:00 zsh
19496 pts/10   00:00:00 sleep
19645 pts/10   00:00:00 ps
```

**Comment by Grimler91** [comment]
That part of the file was rewritten and I thought the patch was not needed anymore. I'll look at re-adding it.

**Comment by macerg** [comment]
Yes, that's fixed it. Thanks.Regards,Richard MaceOn 9 May 2021 02:51, Henrik Grimler ***@***.***> wrote:
Closed #6768 via #6777.
